packaging/debian/postinst

```diff
@@ -58,14 +57,32 @@ case "$1" in
 	    # so that we're in PATH easily
 	    ln -s /var/storagenode/${binary} /usr/bin/${binary}
 	done
-	adduser --system --group --no-create-home --quiet $DAEMON_USER
 
 	if [ "$old_storagedir" != "" ] && [ "$storagedir" != "$old_storagedir" ]; then
 	    echo "New storage directory: ${storagedir} (was ${old_storagedir})"
 	else
 	    echo "Storage directory: ${storagedir}"
 	fi
+
+	# storagenode setup
+	/var/storagenode/storagenode setup \
+	--config-dir "/etc/storagenode" \
+	--identity-dir "$identitydir" \
+	--operator.email "$email" \
+	--operator.wallet "$wallet" \
+	--contact.external-address "$external_addr" \
+	--storage.path "$storagedir" \
+	--storage.allocated-disk-space "$diskspace"
+	
+	# add user and fix ownership
+	adduser --system --group --no-create-home --quiet $DAEMON_USER
+
+	chown -R $DAEMON_USER:$DAEMON_USER /var/storagenode
```

Comment:
Which of those is the most suitable for non static exec?

Reply:
Although if we use `/opt/storj` then we would need to put everything we can under that directory.

> No other package files may exist outside the /opt, /var/opt, and /etc/opt hierarchies except for those package files that must reside in specific locations within the filesystem tree in order to function properly. For example, device lock files must be placed in /var/lock and devices must be located in /dev.

So that's a bit of a pain. I suppose `/var/lib/storj` or `/var/lib/storagenode` might be best after all.